### DISCUSSION TITLE
Dot operator

### DISCUSSION THREAD

**Discussion by Denneisk** [wiremod/wire#2938] [Suggestions]
Add dot `.` operator. Currently I think the only functionality of this would be as an alias for `:`.

**Comment by Divran** [comment]
I'm against having duplicate syntax

**Reply by vurvdev** [reply]
I mean I kinda already broke that tradition with `let`

**Comment by vurvdev** [comment]
Not sure about this, I don't like `:`, but at the same time it's explicit about what it does. Languages having `.` for method syntax and indexing when they are completely different is pretty unintuitive and might catch noobs off guard so I don't really like it.

Also I feel like there's just no point trying to undo `:` since we also have user defined methods syntax.

Maybe it could be done specifically for class methods in the future?
